In [35]:
import os
os.chdir(r"C:\Users\venuk\OneDrive\Desktop\instagram")
os.getcwd()

'C:\\Users\\venuk\\OneDrive\\Desktop\\instagram'

In [36]:
import json
import pandas as pd

In [37]:
data=open('account_history.json',"r")

In [38]:
loaded_data=json.loads(data.read())


In [39]:
loaded_data.keys()

dict_keys(['login_history', 'registration_info'])

In [40]:
len(loaded_data["registration_info"])

6

In [41]:
len(loaded_data["login_history"])

147

In [43]:
loaded_data["registration_info"]

{'registration_username': 'venu',
 'ip_address': '112.133.248.99',
 'registration_time': '2016-08-17T13:14:36+00:00',
 'registration_email': 'venukiran004@gmail.com',
 'registration_phone_number': '',
 'device_name': ''}

In [51]:
loaded_history=loaded_data["login_history"]
loaded_dataFrame=pd.DataFrame(loaded_history)
len(loaded_dataFrame.user_agent.unique())

66

Date of Registration :2016-08-17T13:14:36+00:00
login times=147
unique login devies or browsers=66